cfgov/jinja2/v1/_includes/molecules/featured-menu-content.html

```diff
@@ -0,0 +1,43 @@
+{# ==========================================================================
+
+   featured_menu_content
+
+   ==========================================================================
+
+   Description:
+
+   Create a featured content molecule. See
+   [GHE]/flapjack/Modules-V1/wiki/Featured-Menu-Content
+
+   value: An object with the following options for value.
+
+   value.body:  Body text.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1665/commits/60b8eaf28eafee20484c08edd6d6aeb6fb37dce1